docker/lr-ref/environment.yml

```diff
@@ -0,0 +1,7 @@
+name: lr-align
```

Comment:
this is definitely not right.

Reply:
Oops!  I'll make it the right name.

Reply:
Ah.  This is an image @kvg added when he started this branch.  I'll update the name here.